lib/galaxy/jobs/runners/drmaa.py

```diff
@@ -247,7 +247,7 @@ def check_watched_items( self ):
                 assert external_job_id not in ( None, 'None' ), '(%s/%s) Invalid job id' % ( galaxy_id_tag, external_job_id )
                 state = self.ds.job_status( external_job_id )
             except ( drmaa.InternalException, drmaa.InvalidJobException ) as e:
-                if isinstance( e , drmaa.InvalidJobException ):
+                if isinstance( e, drmaa.InvalidJobException ):
```

Comment:
Should we remove E203 from `setup.cfg` ? I'd pretty happy to fix the 1060 `whitespace before ','` / `whitespace before ':'`